packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -24,12 +24,10 @@ const MainTitleBar = () => {
     const { is_landing_company_loaded, is_switching } = client;
     const { state: wallet_migration_state } = useWalletMigration();
     const { selected_region, handleTabItemClick, toggleRegulatorsCompareModal, content_flag } = traders_hub;
-    const { is_next_wallet_enabled } = useFeatureFlags();
 
     const is_low_risk_cr_real_account =
         content_flag === ContentFlag.LOW_RISK_CR_NON_EU || content_flag === ContentFlag.LOW_RISK_CR_EU;
-    const show_wallets_banner =
-        is_next_wallet_enabled && wallet_migration_state && wallet_migration_state !== 'ineligible';
+    const show_wallets_banner = wallet_migration_state && wallet_migration_state !== 'ineligible';
```

Comment:
Hmmm 🤔 yea, this is the case
Right now it all makes sense but if we ever decide to add a new status for which the banner's not to appear this might be an issue indeed bc the banner will in fact appear

Reply:
Agreed, i've created a PR for changing this https://github.com/binary-com/deriv-app/pull/15348, cannot make changes in this one since it's past QA